setup.cfg

```diff
@@ -21,7 +21,7 @@ input_file = po/server.pot
 output_dir = server/planning/translations
 
 [mypy]
-python_version = 3.8
+python_version = 3.10
```

Comment:
I know Python v3.8 is EOL'd, but this branch is for a bugfix/improvement release for versions that are already in Production. Not sure if we can assume an OS/environment update along with a minor release

Should we instead look at a v2.9 as a change in supported Python version?

Reply:
I've created release/2.9 for it now, so this won't be part of 2.8